textpipe/operation.py

```diff
@@ -70,6 +70,40 @@ def __call__(self, doc):
         return doc.nwords
 
 
+class Words(Operation):
```

Comment:
I'm trying to sneak this in. Noticed it was missing.

Reply:
👍